src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/ApprovalController.java

```diff
@@ -34,9 +34,10 @@ public String postApproval(@PathVariable String companyNumber,
 
         try {
             transactionService.closeTransaction(transactionId);
-        } catch (ApiErrorResponseException e) {
-            // TODO: handle ApiErrorResponseExceptions (SFA-594)
+        } catch (ServiceException e) {
+
             LOGGER.errorRequest(request, "Failed to close transaction", e);
+            return "error";
```

Comment:
Could we have a protected string in the base controller to hold the error view?

Reply:
I've added a suitable string constant to the base controller and replaced the string literals in the controllers with this.